tests/integration/conftest.py

```diff
@@ -100,14 +100,20 @@ def test_mytest(client_toggled):
     """
     params = request.node.get_marker('config_toggle')
 
+    old_configs = {}
     for parameter, value in params.kwargs['parameters'].items():
+        old_configs[parameter] = app.config.get(parameter)
         app.config[parameter] = value
 
     with app.test_client() as client:
         yield client
 
-    # reset app config just in case
-    app.config.from_object("sheepdog.test_settings")
+    # reset app config to the original state
```

Comment:
discussed this offline: plz use test_settings as a state to revert to, not just the previous one